Makefile

```diff
@@ -0,0 +1,48 @@
+.PHONY: help clean-pyc clean-build isort lint test docker-build
```

Comment:
For me this makefile is just one level of indirection to much. Why don't you just call tox directly in gitlab actions?

Reply:
Yes and no. For me tox is just a test manager, while makefile may contain way more helpful commands (also wrapping tox commands). This allows me to standardize commands, regardless of what I use behind, and also saves me (and others) from digging through "helper" shell scripts.